sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class InterestRateSwaptionExamples: DemoInstrumentBase
+    {
+        internal override void CreateAndUpsertMarketDataToLusid(string scope, ModelSelection.ModelEnum model, LusidInstrument instrument)
```

Comment:
Consider adding a comment or TODO for why this and GetAndValidatePortfolioCashFlows is empty